js/layout-javascript/news-feed-code.js

```diff
@@ -1112,9 +1112,7 @@ DynamicList.prototype.initialize = function() {
         container: _this.$container,
         records: records
       }).then(function() {
-        return Fliplet.User.getCachedSession();
-      }).then(function(session) {
-        _this.Utils.Records.setSource(session, _this.data);
+        return _this.Utils.Records.setFilterValues(_this.data);
```

Comment:
Use an embedded options object so that we can add more options to `_this.Utils.Records.setFilterValues()` if necessary without having to alter `_this.data`.

```suggestion
        return _this.Utils.Records.setFilterValues({
          config: _this.data
        });
```